/** Publications data. */
const publications = [
	{
		title: '20 years of the Bio-Analytic Resource for Plant Biology',
		authors: [
			'Alexander Sullivan',
			'Michael N Lombardo',
			'Asher Pasha',
			'Vincent Lau',
			'Jian Yun Zhuang',
			'Ashley Christendat',
			'Bruno Pereira',
			'Tianhui Zhao',
			'Youyang Li',
			'Rachel Wong',
			'Faisal Z Qureshi',
			'Nicholas J Provart',
		],
		abstract:
			'The Bio-Analytic Resource for Plant Biology (‘the BAR’, at https://bar.utoronto.ca) is celebrating its 20th year in operation in 2025. The BAR encompasses and provides visualization tools for large ‘omics data sets from plants. The BAR covers data from Arabidopsis, tomato, wheat, barley and 29 other plant species (with data for 2 others to be released soon). These data include nucleotide and protein sequence data, gene expression data, protein-protein and protein–DNA interactions, protein structures, subcellular localizations, and polymorphisms. The data are stored in more than 200 relational databases holding 186 GB of data and are presented to the researchers via web apps. These web apps provide data analysis and visualization tools. Some of the most popular tools are eFP (‘electronic fluorescent pictograph’) Browsers, ePlants and ThaleMine (an Arabidopsis-specific instance of InterMine). The BAR was designated a Global Core Biodata Resource in 2023. Like other GCBRs, the BAR has excellent operational stability, provides access without login requirement, and provides an API for researchers to be able to access BAR data programmatically. We present in this update a new overarching search tool called Gaia that permits easy access to all BAR data, powered by machine learning and artificial intelligence.',
		doi: '10.1093/nar/gkae920',
		journal: 'Nucleic Acids Research',
		date: '2024-10-23',
		'related-project': 'gaia',
	},
	{
		title: 'ePlant in 2021: New Species, Viewers, Data Sets, and Widgets',
		authors: [
			'Ben Waese-Perlman',
			'Asher Pasha',
			'Chantal Ho',
			'Amirahmad Azhieh',
			'Yushan Liu',
			'Alexander Sullivan',
			'Vincent Lau',
			'Eddi Esteban',
			'Jamie Waese',
			'George Ly',
			'Cornelia Hooper',
			'S. Evan Staton',
			'Nicholas Brereton',
			'Cuong Le',
			'Rex Nelson',
			'Shelley Lumba',
			'David Goodstein',
			'A. Harvey Millar',
			'Isobel Parkin',
			'Lewis Lukens',
			'Juergen Ehlting',
			'Loren Rieseberg',
			'Frédéric Pitre',
			'Anne Brown',
			'Nicholas J. Provart',
		],
		abstract:
			'ePlant was introduced in 2017 for exploring large Arabidopsis thaliana data sets from the kilometre to nanometre scales. In the past four years we have used the ePlant framework to develop ePlants for 15 agronomically-important species: maize, poplar, tomato, Camelina sativa, soybean, potato, barley, Medicago truncatula, eucalyptus, rice, willow, sunflower, Cannabis sativa, wheat and sugarcane. We also updated the interface to improve performance and accessibility, and added two new views to the Arabidopsis ePlant - the Navigator and Pathways viewers. The former shows phylogenetic relationships between homologs in other species and their expression pattern similarities, with links to view data for those genes in the respective ePlants. The latter shows Plant Reactome metabolic reactions. We also describe new Arabidopsis data sets including single cell RNA-seq data from roots, and how to embed ePlant eFP expression pictographs into any web page.',
		doi: '10.1101/2021.04.28.441805',
		journal: 'Preprint (bioRxiv)',
		date: '2021-04-29',
		'related-project': 'eplant-plant-efp',
	},
	{
		title: "An 'eFP-Seq Browser' for visualizing and exploring RNA sequencing data",
		authors: [
			'Alexander Sullivan',
			'Priyank Purohit',
			'Nowlan H. Freese',
			'Asher Pasha',
			'Eddi Esteban',
			'Jamie Waese',
			'Alison Wu',
			'Michelle Chen',
			'Chih Ying Chin',
			'Richard Song',
			'Snehal R. Watharkar',
			'Agnes P. Chan',
			'Vivek Krishnakumar',
			'Matthew W. Vaughn',
			'Chris Town',
			'Ann E. Loraine',
			'Nicholas J. Provart',
		],
		abstract:
			"Improvements in next-generation sequencing technologies have resulted in dramatically reduced sequencing costs. This has led to an explosion of '-seq'-based methods, of which RNA sequencing (RNA-seq) for generating transcriptomic data is the most popular. By analysing global patterns of gene expression in organs/tissues/cells of interest or in response to chemical or environmental perturbations, researchers can better understand an organism's biology. Tools designed to work with large RNA-seq data sets enable analyses and visualizations to help generate hypotheses about a gene's function. We present here a user-friendly RNA-seq data exploration tool, called the 'eFP-Seq Browser', that shows the read map coverage of a gene of interest in each of the samples along with 'electronic fluorescent pictographic' (eFP) images that serve as visual representations of expression levels. The tool also summarizes the details of each RNA-seq experiment, providing links to archival databases and publications. It automatically computes the reads per kilobase per million reads mapped expression-level summaries and point biserial correlation scores to sort the samples based on a gene's expression level or by how dissimilar the read map profile is from a gene splice variant, to quickly identify samples with the strongest expression level or where alternative splicing might be occurring. Links to the Integrated Genome Browser desktop visualization tool allow researchers to visualize and explore the details of RNA-seq alignments summarized in eFP-Seq Browser as coverage graphs. We present four cases of use of the eFP-Seq Browser for ABI3, SR34, SR45a and U2AF65B, where we examine expression levels and identify alternative splicing. The URL for the browser is https://bar.utoronto.ca/eFP-Seq_Browser/. OPEN RESEARCH BADGES: This article has earned an Open Data Badge for making publicly available the digitally-shareable data necessary to reproduce the reported results. Tool is at https://bar.utoronto.ca/eFP-Seq_Browser/; RNA-seq data at https://s3.amazonaws.com/iplant-cdn/iplant/home/araport/rnaseq_bam/ and https://s3.amazonaws.com/iplant-cdn/iplant/home/araport/rnaseq_bam/Klepikova/. Code is available at https://github.com/BioAnalyticResource/eFP-Seq-Browser.",
		doi: '10.1111/tpj.14468',
		journal: 'The Plant Journal',
		date: '2019-07-26',
		'related-project': 'efp-seq-browser',
	},
];

export default publications;
